src/components/ChangeLog/ChangeLog.vue

```diff
@@ -13,7 +13,9 @@ const { closeModal, isModalOpen, openModal } = useModal();
 const html = ref({ header: '', body: '' });
 
 function fetchChangeLog() {
-  return fetch('/CHANGELOG.md')
+  const scriptEl = document.head.querySelector('script[src]') as HTMLScriptElement;
+  const buildString = scriptEl?.src?.match(/\/index-(\w+)\.js/)?.[1];
+  return fetch(`/CHANGELOG.md?build=${buildString}`)
```

Comment:
Oh Nice! 💯 
I thought about using Vite build process to add the hash, but this works